tests/integration/nm/ovs_test.py

```diff
@@ -406,3 +432,19 @@ def test_create_vlan_over_ovs_system_interface_bond_with_same_name(
                 ]
             }
         )
+
+
+@pytest.mark.tier1
+def test_modify_state_with_ovs_bridge_interface_with_ipv4_dns(
```

Comment:
The test does not make it clear that the duplicate interfaces need to be reported by the NM plugin in a certain order to trigger the error. Ideally, this should be enforced to ensure that the actual error condition is tested.

Reply:
I've changed the test name, please could you re-check it? Thanks!